### ISSUE TITLE
[New Feature] Health format as 16579 not 16.5k

### ISSUE THREAD

**Issue by Denziz** [ElvUI-TBC/ElvUI#420]
Hi! My feature request is about health formatting, currently what I can see is that there are 6 different options for health formatting.

[health:current]
[health:percent]
[health:current-max]
[health:current-max-percent]
[health:deficit]

Although I would really love if there was a [health:flat] option which does not shorten the text by adding a K to it, so for example atm [health:current] shows 16.9k (decimal depends how many you have set in General -> Decimal Length) and instead it would look like this, 16975. 

TLDR: Option to show the health numbers flat without any shortening.

Thanks in advance, hope you have a great day, Kasper.

**Comment by Denziz** [comment]
I'm silly, just found out by doing a google search there are hidden health formats within ElvUI, [curhp] does the work just fine!
